packages/paypal-commerce-utils/src/paypal-commerce-sdk.spec.ts

```diff
@@ -15,10 +15,17 @@ describe('PayPalCommerceSdk', () => {
     let paymentMethod: PaymentMethod;
     let paypalAxoSdk: PayPalAxoSdk;
     let subject: PayPalCommerceSdk;
+
     const paypalMessagesSdk: PayPalMessagesSdk = {
         Messages: jest.fn(),
     };
 
+    Object.defineProperty(globalThis, 'crypto', {
```

Comment:
Instead of globalThis, did you try window here?

Reply:
I didn't try it before. But it works as well